docker/hive/start-hive.sh

```diff
@@ -50,6 +60,34 @@ cat >${HIVE_HOME}/conf/hive-site.xml <<EOL
         <name>fs.default.name</name>
         <value>${DEFAULT_FS}</value>
      </property>
+     <property>
+        <name>hive.metastore.sasl.enabled</name>
+        <value>false</value>
+     </property>
+     <property>
+        <name>fs.s3a.impl</name>
+        <value>org.apache.hadoop.fs.s3a.S3AFileSystem</value>
+     </property>
+     <property>
+        <name>fs.s3.impl</name>
+        <value>org.apache.hadoop.fs.s3a.S3AFileSystem</value>
+     </property>
+     <property>
```

Comment:
Can you make these 2 properties optional?

Reply:
@lyogev how exactly? if they're empty, we'll get empty entries